### PR TITLE
SKME Zeno Via compat

### PR DESCRIPTION
**Pull request by paulgali**
## Description

Added Zeno to Via v3

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/20023

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by paulgali** [comment]
Original file was from mid 2022. This has been deleted and a new v3 compat file has been added instead. Thanks.

File: v3/skme/ZenoErgo/Zeno004.json [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+      "labels": [
+        "Full Backspace",
+        "Full Right Shift",
+        "Long Left Space (2.75u)",
+        "Short Right Space (2u)",
+        "1.25u Bottom Row"
+      ],
+      "keymap": [
+  {
+    "name": "zenokle"
```
Comment (Xelus22) [inline v3/skme/ZenoErgo/Zeno004.json:19]: Please remove this

Comment (wilba) [reply v3/skme/ZenoErgo/Zeno004.json:19]: how did this get past validation script? @ValdisThomann

Comment (paulgali) [reply v3/skme/ZenoErgo/Zeno004.json]: My bad, this is removed.

**Comment by Xelus22** [comment]
Hi for future PR's please paste in the correct QMK PR. I changed your original link from 20003 -> 20023.
Thanks!

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**